### PR TITLE
Move glossary to the end of the spec

### LINKED ISSUE #362: Move glossary to the end of the spec
Traditionally, a glossary comes at the end of a piece of writing/book. Currently, we have it right at the start, which is a bit unusual. I'd propose moving it to the end, after "Converting between `tiltX`/`tiltY` ..."

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes https://github.com/w3c/pointerevents/issues/362

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/pull/363.html" title="Last updated on Apr 21, 2021, 10:35 AM UTC (6bb3817)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/363/5565d23...6bb3817.html" title="Last updated on Apr 21, 2021, 10:35 AM UTC (6bb3817)">Diff</a>

### CODE REVIEW

**Comment by patrickhlauke** [comment]
would appreciate a pair of eyes whenever your get a chance (as can't merge without one approving review) - this should be fairly non-controversial

**Comment by smaug----** [comment]
I'm not sure we should make this change.

File: index.html [hunk 1]
```diff
@@ -1333,6 +1301,37 @@ <h2>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
...
+            <dt><dfn>hit test</dfn></dt>
+                <dd>The process by which a user agent determines a target element for a pointer event. Typically, this is determined by considering the pointer's location and also the visual layout of elements in a document on screen media.</dd>
+            <dt><dfn>pointer</dfn></dt>
+                <dd>A hardware agnostic representation of input devices that can target a specific coordinate (or set of coordinates) on a screen, such as a mouse, pen, or touch contact.</dd>
+            <dt><dfn>user agent</dfn></dt>
+                <dd>A program, such as a browser or content authoring tool, normally running on a client machine, which acts on a user's behalf in retrieving, interpreting, executing, presenting, or creating content.</dd>
+        </dl>
+    </section>
     <section>
         <h2>Security and privacy considerations</h2>
```
Comment (smaug----) [inline index.html]: I wonder if glossary should be after 'Security and privacy considerations'

**Review by smaug---- — APPROVED** [review]
I guess I don't care too much where glossary is :)

**Comment by patrickhlauke** [comment]
admittedly, i'm not overly precious about it either...but i do feel it helps a first-time reader get into the spec's actual stuff more easily. thanks @smaug----

File: index.html (hunk 1 above)
Comment (patrickhlauke) [reply index.html]: ah, good point. yes, that make even more sense. will move it there and then merge

**Pull request merged**